lib/http4s-lambda-handler/src/main/scala/com/gu/http4s/Http4sLambdaHandler.scala

```diff
@@ -110,14 +109,14 @@ class Http4sLambdaHandler(service: HttpRoutes[IO]) {
         }
         .toList
 
-    Uri(path = apiGateWayRequest.path, query = Query(queryStringValues: _*))
+    Uri(path = Uri.Path.unsafeFromString(apiGateWayRequest.path), query = Query(queryStringValues: _*))
   }
 
   private def extractHeaders(apiGateWayRequest: LambdaRequest): Headers = {
-    Headers.of(
+    Headers(
       apiGateWayRequest.multiValueHeaders
         .getOrElse(Nil)
-        .flatMap { case (key, multiValue) => multiValue.map(value => Header(key, value)) }
+        .flatMap { case (key, multiValue) => multiValue.map[Header.ToRaw](value => (key, value)) }
         .toList: _*,
     )
```

Comment:
`Headers.apply`  seems to take the values as a Header.ToRaw which is the target of an implicit conversion from various other kinds of value, so I've leveraged that.  The other option (to avoid the implicit conversion) would be to build a `Header.Raw` myself, and pass in to `new Headers`
more like this
```suggestion
    new Headers(
      apiGateWayRequest.multiValueHeaders
        .getOrElse(Nil)
        .flatMap { case (key, multiValue) => multiValue.map(value => Header.Raw(CIString(key), value)) }
        .toList: _*,
    )
```